<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordWidget</class>
 <widget class="QWidget" name="PasswordWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>303</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">PasswordWidget</string>
  </property>
  <layout class="QVBoxLayout" name="pVerticalLayout_">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Set Password</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="pLabelOldPassword_">
        <property name="text">
         <string notr="true">&amp;Old Password:</string>
        </property>
        <property name="buddy">
         <cstring>pLineEditOldPassword_</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="pLineEditOldPassword_">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="pLabelNewPassword_">
        <property name="text">
         <string notr="true">&amp;New Password:</string>
        </property>
        <property name="buddy">
         <cstring>pLineEditNewPassword_</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="pLineEditNewPassword_">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="pLabelConfirmNewPassword_">
        <property name="text">
         <string notr="true">Confirm New &amp;Password:</string>
        </property>
        <property name="buddy">
         <cstring>pLineEditConfirmNewPassword_</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="pLineEditConfirmNewPassword_">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="pLabelNewPasswordHint_">
        <property name="text">
         <string notr="true">New Password &amp;Hint:</string>
        </property>
        <property name="buddy">
         <cstring>pLineEditNewPasswordHint_</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="pLineEditNewPasswordHint_"/>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QCheckBox" name="pCheckBoxShowCharacters_">
        <property name="text">
         <string>Show &amp;Characters</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <spacer name="pVerticalSpacerPasswordHint_">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="7" column="0">
       <widget class="QPushButton" name="pPushButtonShowRequirements_">
        <property name="text">
         <string>Show &amp;Requirements</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Validation</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="pLabelPasswordInfo_">
        <property name="text">
         <string notr="true">infoLine1
infoLine2</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="::cppdevtk::gui::CapsLockWidget" name="pCapsLockWidget_" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>::cppdevtk::gui::CapsLockWidget</class>
   <extends>QWidget</extends>
   <header location="global">cppdevtk/gui/caps_lock_widget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pLineEditOldPassword_</tabstop>
  <tabstop>pLineEditNewPassword_</tabstop>
  <tabstop>pLineEditConfirmNewPassword_</tabstop>
  <tabstop>pCheckBoxShowCharacters_</tabstop>
  <tabstop>pLineEditNewPasswordHint_</tabstop>
  <tabstop>pPushButtonShowRequirements_</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
